src/sqlancer/MainOptions.java

```diff
@@ -50,6 +50,27 @@ public class MainOptions {
     @Parameter(names = "--print-failed", description = "Logs failed insert, create and other statements without results", arity = 1)
     private boolean loggerPrintFailed = true; // NOPMD
 
+    @Parameter(names = "--log-execution-result", description = "Logs the execution result of each statement (requires --log-each-select to be enabled)", arity = 1)
+    private boolean logExecutionResult = true; // NOPMD
+
+    @Parameter(names = "--log-query-plan", description = "Logs the query plans of each statemen", arity = 1)
+    private boolean logQueryPlan; // NOPMD
+
+    @Parameter(names = "--enable-qpg", description = "Enable the experimental feature Query Plan Guidance (QPS)", arity = 1)
```

Comment:
Perhaps we can use a common prefix for all QPG options? Here, we could just use `--qpg` as an option (or `--qpg-enable`).